src/renderer/components/cluster-icon/cluster-icon.tsx

```diff
@@ -29,12 +33,31 @@ const defaultProps: Partial<Props> = {
 export class ClusterIcon extends React.Component<Props> {
   static defaultProps = defaultProps as object;
 
+  @observable eventCount = 0;
+
+  get eventCountBroadcast() {
+    return `cluster-warning-event-count:${this.props.cluster.id}`;
+  }
+
+  componentDidMount() {
+    const subscriber = subscribeToBroadcast(this.eventCountBroadcast, (ev, eventCount) => {
+      if (eventCount != this.eventCount) {
+        this.eventCount = eventCount;
+      }
```

Comment:
The outcome seems to be the same as:
```suggestion
      this.eventCount = eventCount;
```
so why the check?

Reply:
Fixed.